nixos/modules/security/pam.nix

```diff
@@ -338,7 +338,7 @@ let
           auth required pam_deny.so
 
           # Password management.
-          password requisite pam_unix.so nullok sha512
+          password sufficient pam_unix.so nullok sha512
```

Comment:
can you add a line or two below my release notes about `pam_unix` in `nixos/doc/manual/release-notes/rl-1903.xml`, explaining the change?